libs/build-nss-desktop.sh

```diff
@@ -22,117 +22,88 @@ if [ -n "${CROSS_COMPILE_TARGET}" ] && [ "$(uname -s)" != "Linux" ]; then
 fi
 
 if [[ "${CROSS_COMPILE_TARGET}" =~ "win32-x86-64" ]]; then
-  NSS_DIR=$(abspath "desktop/win32-x86-64/nss")
+  DIST_DIR=$(abspath "desktop/win32-x86-64/nss")
 elif [[ "${CROSS_COMPILE_TARGET}" =~ "darwin" ]]; then
-  NSS_DIR=$(abspath "desktop/darwin/nss")
+  DIST_DIR=$(abspath "desktop/darwin/nss")
 elif [ -n "${CROSS_COMPILE_TARGET}" ]; then
   echo "Cannot build NSS for unrecognized target OS ${CROSS_COMPILE_TARGET}"
   exit 1
 elif [ "$(uname -s)" == "Darwin" ]; then
-  NSS_DIR=$(abspath "desktop/darwin/nss")
+  DIST_DIR=$(abspath "desktop/darwin/nss")
 elif [ "$(uname -s)" == "Linux" ]; then
   # This is a JNA weirdness: "x86-64" rather than "x86_64".
-  NSS_DIR=$(abspath "desktop/linux-x86-64/nss")
+  DIST_DIR=$(abspath "desktop/linux-x86-64/nss")
 else
    echo "Cannot build NSS on unrecognized host OS $(uname -s)"
    exit 1
 fi
 
-if [ -d "${NSS_DIR}" ]; then
-  echo "${NSS_DIR} folder already exists. Skipping build."
+if [ -d "${DIST_DIR}" ]; then
+  echo "${DIST_DIR} folder already exists. Skipping build."
   exit 0
 fi
 
-NSPR_BUILD_DIR=$(mktemp -d)
-BUILD_DIR=$(mktemp -d)
-
-EXTRA_MAKE_ARGS=()
-# Build NSPR.
-pushd "${NSPR_BUILD_DIR}"
+# TODO compile on macOS/windows machines once `chainOfTrust` is supported on macOS (1499051).
```

Comment:
follow up issue?

Reply:
Ye, I'll make a list of all the follow-up issues